src/stryker/quickstart.pug

```diff
@@ -18,52 +18,59 @@ block content
-                code 
-                    | npm install --save-dev stryker stryker-api
             :markdown-it
-                If you choose to not install the stryker-cli, use `./node_modules/.bin/stryker` instead of `stryker` for the rest of the quickstart.
+                The easiest way to get started with Stryker is by installing the stryker-cli globally. 
+                It is a small package which forwards commands to your local stryker instance. 
+                            
+                ```shell
+                npm install -g stryker-cli
+                ```
+
+                Optionally, you could also install stryker directly yourself.
+                
+                ```shell
+                npm install --save-dev @stryker-mutator/core
+                ```
+                
+                If you choose to not install the stryker-cli, use `npx stryker` instead of `stryker` for the rest of the quickstart.
             hr/
     section.row
         .col-md-12
             h3 #[span.badge.badge-primary 3] Configure
-            p Run this command the configure stryker.
-            pre.stryker-install
-                code 
-                    | stryker init
-            p If you're asked to install stryker, choose yes. Make decisions for the rest of your questions:
-            ol
-                li Which test runner do you want to use?
-                li Which test framework do you want to use?
-                li Which reporter(s) do you want to use?
-            p Please let us know if your option is missing here.
-            
+
             :markdown-it
+                Run this command the configure stryker.
+                
+                ```shell
+                stryker init
+                ```
+
+                If you're asked to install stryker, choose **Yes**. Follow the questionnaire.
+                
+                Please let us know if your option is missing here [by opening an issue](https://github.com/stryker-mutator/stryker/issues).
+
                 After the init is done, inspect the `stryker.conf.js` file. 
-                Update the `files` and `mutate` properties if appropriate.
-                For more information on what these options mean, take a look at the [Stryker readme](https://github.com/stryker-mutator/stryker/tree/master/packages/stryker#readme)
+                
+                For more information on what these options mean, take a look at the [Stryker readme](https://github.com/stryker-mutator/stryker/tree/master/packages/core#readme)
             hr/
     section.row
         .col-md-12
             h3 #[span.badge.badge-primary 4] Let's kill some mutants
-            p Run stryker to mutation test your project
-            pre.stryker-install
-                code 
-                    | stryker run
+
+            :markdown-it
+                Run stryker to mutation test your project
+                
+                ```shell
+                stryker run
```

Comment:
`shell` syntax highlighting should start with a `$`. Otherwise it should be `bash` or `sh`

Reply:
Thanks!